barricade-annotations/src/main/java/com/mutualmobile/barricade/response/BarricadeResponseSet.java

```diff
@@ -10,9 +10,11 @@
 public class BarricadeResponseSet {
   public List<BarricadeResponse> responses;
   public int defaultIndex;
```

Comment:
For readability, I think it'll be better to have "default" in both fields. `currentIndex` doesn't really communicate what it is for. The reason I used "original" is that it denotes that the other value would be changed at some point and it fits nicely with the concept of "resetting" the config.